src/_guides/libraries/private-files.md

```diff
@@ -27,9 +26,7 @@ created by pub and dart2js:
 build/
 packages/
```

Comment:
Should probably get rid of this. Users have to explicitly opt-in to create `packages/` instead of `.packages`

Reply:
Done